packages/js/src/ui/components/Inbox.tsx

```diff
@@ -87,13 +87,16 @@ export const InboxContent = (props: InboxContentProps) => {
 
   return (
     <div
-      class={style(
-        'inboxContent',
-        cn('nt-h-full nt-flex nt-flex-col [&_.nv-preferencesContainer]:nt-pb-8 [&_.nv-notificationList]:nt-pb-8', {
-          '[&_.nv-preferencesContainer]:nt-pb-12 [&_.nv-notificationList]:nt-pb-12': isDevelopmentMode(),
-          '[&_.nv-preferencesContainer]:nt-pb-8 [&_.nv-notificationList]:nt-pb-8': !isDevelopmentMode(),
-        })
-      )}
+      class={style({
```

Comment:
I've updated the params of the `style` callback, that's why you would see a lot of similar changes in this PR